packages/router-api/src/msg.rs

```diff
@@ -25,12 +25,23 @@ pub enum ExecuteMsg {
      * Router Admin Methods
      * All the below messages should only be called by the router admin
      */
-    // Freezes a chain, in the specified direction.
+    /// Freezes a chain, in the specified direction.
     FreezeChain {
         chain: ChainName,
         direction: GatewayDirection,
     },
-    // Unfreezes a chain, in the specified direction.
+    /// If no filter is provided, freezes all chain connections in every direction. Should only be called in emergencies.
+    /// If the filter is set, only the specified chains will be frozen.
+    FreezeAllChains { chains: Option<Vec<ChainName>> },
+
+    /// Unfreezes the specified chains in the specified directions. In contrast to FreezeAllChains,
+    /// this does not allow unfreezing all chains, because that command is too dangerous.
+    /// It would make it too easy to accidentally unfreeze a chain that should not be unfrozen.
+    UnfreezeChains {
+        chains: Vec<(ChainName, GatewayDirection)>,
```

Comment:
Why not remove `FreezeChain` and `UnfreezeChain` in favor of these versions that take a list of chains? You can always pass a list of length 1, and now I feel the API is larger than it needs to be

Reply:
Sorry, I mean why not have `FreezeChains`, `UnfreezeChains` and `FreezeAllChains` as the API?